mocha/config/protractor-chrome.config.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+const GLOBAL_TIMEOUT = 40e3;
+
+exports.config = {
+  specs: "../specs/test.spec.js",
```

Comment:
Are you able to execute the test2.spec.js file? Please fix it in all file.

Reply:
done